### PR TITLE
ndp: move type definitions out of ng_ context

### PR DESCRIPTION
**Pull request by authmillenon**
Depends on #3599 and all its dependencies.

### CODE REVIEW

**Comment by authmillenon** [comment]
Rebased to current #3599.

**Comment by authmillenon** [comment]
Reassign at will.

**Comment by authmillenon** [comment]
Rebased to current #3599.

**Comment by authmillenon** [comment]
Rebased to current #3599.

**Comment by OlegHahm** [comment]
Please rebase.

**Comment by authmillenon** [comment]
Rebased.

File: sys/include/net/ndp.h [hunk 1]
```diff
@@ -17,8 +17,8 @@
  *
  * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
  */
-#ifndef NG_NDP_TYPES_H_
-#define NG_NDP_TYPES_H_
+#ifndef NDP_
+#define NDP_
```
Comment (OlegHahm) [inline sys/include/net/ndp.h]: _NDP_H_

Comment (authmillenon) [inline sys/include/net/ndp.h]: Addressed.

**Comment by OlegHahm** [comment]
You can squash immediately.

**Comment by authmillenon** [comment]
Squashed.

**Comment by OlegHahm** [comment]
ACK

**Pull request merged**